src/index.js

```diff
@@ -213,7 +334,12 @@ class ClientAuthOIDC {
    * @return {Promise<string>} Resolves to the logged in user's WebID URI
    */
   login (providerUri) {
-    this.clearCurrentUser()
+    if (this.webId) {
```

Comment:
I dunno about this. What about when credentials expire, and you have to re-log in?